src/components/StackedChart/index.js

```diff
@@ -0,0 +1,317 @@
+import dayjs from "dayjs";
+import PropTypes from "prop-types";
+import React, { useCallback, useEffect, useState } from "react";
+import {
+  AreaChart,
+  Area,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ResponsiveContainer,
+  Bar,
+  ComposedChart,
+  Legend,
+} from "recharts";
+
+import { SupportedNetwork } from "../../constants";
+import { formattedNum, formattedPercent } from "../../utils";
+import Header from "./Header";
+import CrosshairTooltip from "./CrosshairTooltip";
+
+const TIME_FILTER_OPTIONS = {
+  MONTH_1: "1M",
+  MONTH_3: "3M",
+  YEAR: "1Y",
+};
+const NETWORK_COLORS = {
+  [SupportedNetwork.MAINNET]: "#2974e0",
+  [SupportedNetwork.XDAI]: "#4526A2",
+  [SupportedNetwork.ARBITRUM_ONE]: "#feb125",
+};
+
```

Comment:
```suggestion
const LegendItem = (value) => (
  <TYPE.light color="text1" as="span">
    {value}
  </TYPE.light>
);
```

Reply:
theme aware components are better since once we have dark, light mode switch it will work by default.

Reply:
I guess we might not have themes in dxstats. If we have it will be a full blown redesign. :) Not in 2022 for sure. :)